src/emcommon/diary/base_mode_colors.py

```diff
@@ -0,0 +1,43 @@
+modeColors = {
```

Comment:
```suggestion
mode_colors = {
```
Use snake_case for python code